data/json/monsters/feral_humans.json

```
@@ -1019,5 +1019,140 @@
       "PATH_AVOID_DANGER_1",
       "DROPS_AMMO"
     ]
+  },
+  {
+    "id": "mon_feral_big_pappy",
+    "type": "MONSTER",
+    "name": { "str": "Big Pappy" },
+    "//": "Boss enemy for Nursing Home special",
+    "description": "A gigantic, monstrous old man with a crazed, anguished face. Created by a twisted science experiment. Carries a large cane, and seems intent to discipline you with it.",
```

Two sentence fragments bad
```suggestion
    "description": "A gigantic, monstrous old man with a crazed, anguished face created by a twisted science experiment. He carries a large cane, and seems intent to discipline you with it.",
```